src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -77,7 +77,8 @@ public void addPerson(Person p) {
     /**
      * Replaces the given person {@code target} in the list with {@code editedPerson}.
      * {@code target} must exist in the address book.
-     * The person identity of {@code editedPerson} must not be the same as another existing person in the address book.
+     * The person identity of {@code editedPerson} must not be the same as another existing person
```

Comment:
While this comment is informative, perhaps a better way to word it would be 'The identity of {@code editedPerson} should not be the same as another existing {@code Person}'?
Also, the context needs to be changed, because we are changing this from the Address Book to the ORGANice Transplant Manager